docs/2.Routes_and_controllers.md

```diff
@@ -25,6 +23,14 @@ You must create your controller following two rules:
     
     }
     
+If you want to use namespaces in your controller you should modify you composer.json to autoload your namespaces. For example, in compose.json add:
+    
+    "extra": {
+        ...
+    },
+    "autoload": {
+        "psr-4": {"MyCompany\\myPlugin\\": "public/wp-content/plugins/my-cool-plugin/"}
```

Comment:
Each plugin has its composer.json file where you need to configure its PSR. 
In the general composer.json you can add new autoload namespaces outside of the logic of WordPress.
In fact that could be very interesting to explain in the documentation.

I´m going to take this pull request as an idea in order to improve the documentation.